pandas/core/indexes/datetimes.py

```diff
@@ -1922,6 +1925,41 @@ def tz_convert(self, tz):
         ------
         TypeError
             If DatetimeIndex is tz-naive.
+
+        See Also
+        --------
+        tz_localize : Localize tz-naive DatetimeIndex to given time zone
+            (using pytz/dateutil), or remove timezone from tz-aware
+            DatetimeIndex.
+
+        Examples
+        --------
+        >>> didx = pd.DatetimeIndex
+        (start='2014-08-01 09:00', freq='H', periods=10, tz='Europe/Berlin')
+
+        >>> didx
+        DatetimeIndex(['2014-08-01 09:00:00+02:00','2014-08-01 10:00:00+02:00',
+        '2014-08-01 11:00:00+02:00', '2014-08-01 12:00:00+02:00',
+        '2014-08-01 13:00:00+02:00', '2014-08-01 14:00:00+02:00',
+        '2014-08-01 15:00:00+02:00', '2014-08-01 16:00:00+02:00',
+        '2014-08-01 17:00:00+02:00', '2014-08-01 18:00:00+02:00'],
+        dtype='datetime64[ns, Europe/Berlin]', freq='H')
+
+        >>> didx.tz_convert('US/Eastern')
+        DatetimeIndex(['2014-08-01 03:00:00-04:00','2014-08-01 04:00:00-04:00',
+        '2014-08-01 05:00:00-04:00', '2014-08-01 06:00:00-04:00',
+        '2014-08-01 07:00:00-04:00', '2014-08-01 08:00:00-04:00',
+        '2014-08-01 09:00:00-04:00', '2014-08-01 10:00:00-04:00',
+        '2014-08-01 11:00:00-04:00', '2014-08-01 12:00:00-04:00'],
+        dtype='datetime64[ns, US/Eastern]', freq='H')
+
+        >>> didx.tz_convert(None)
```

Comment:
Can you put here a small sentence explaining the example?

Reply:
@jorisvandenbossche didn't realize we actually allow None here.